test/scrabble_test.rb

```diff
@@ -5,9 +5,121 @@
 require 'pry'
 
 class ScrabbleTest < Minitest::Test
+
+  def test_that_tally_is_default_value_of_zero
+    assert_equal 0, Scrabble.new.tally
+  end
+
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_two_letters
```

Comment:
Really like how robust this test suite is. Since you've taken the time to break the logic in your class into supporting methods, would like to see you test those methods directly with unit tests. Currently looks like all integration.